frontend/wallet/package.json

```diff
@@ -8,7 +8,7 @@
     "bs-electron": "github:Schmavery/bs-electron",
     "graphql_ppx": "^0.2.8",
     "reason-apollo": "^0.15.2",
-    "reason-react": "^0.5.3"
+    "reason-react": "github:reasonml/reason-react"
```

Comment:
pin it to a commit (but remember to bump it?)

Reply:
actually master is broken right now, that's why CI is failing lol